packages/components/src/combobox-control/test/index.tsx

```diff
@@ -306,4 +306,153 @@ describe.each( [
 		expect( onChangeSpy ).toHaveBeenCalledWith( targetOption.value );
 		expect( input ).toHaveValue( targetOption.label );
 	} );
+
+	it( 'should render with Reset button disabled', () => {
+		render(
+			<Component
+				options={ timezones }
+				label={ defaultLabelText }
+				allowReset
+			/>
+		);
+
+		const resetButton = screen.getByRole( 'button', { name: 'Reset' } );
+
+		expect( resetButton ).toBeInTheDocument();
+		expect( resetButton ).toBeVisible();
```

Comment:
```suggestion
		expect( resetButton ).toBeVisible();
```

`toBeVisible()` should already imply that the element is in the document

Reply:
Yup, I just copied and pasted from the existing test `should render with visible label`, will fix it there as well